state/lease/client.go

```diff
@@ -177,6 +178,9 @@ func (client *client) ExpireLease(name string) error {
 // Refresh is part of the Client interface.
 func (client *client) Refresh() error {
 	client.logger.Tracef("refreshing")
+	if wrench.IsActive("lease", "refresh") {
```

Comment:
This leads to a lot of DEBUG level complaints about wrench directory not existing.
Is it worth the effort to add it?

Reply:
Worth the effort to add what? We can remove it, but it seems to me the right thing to do is change wrench to use the TRACE level.

Reply:
I agree with @axw on that point.

Reply:
@axw care to do a simple follow up branch that moves the wrench package Debug statements to Trace?

Reply:
yup, will do today